<launch>

    <group>
        <remap from="generic_setpoint" to="ang_setpoint"/>
        <remap from="current_state" to="ang_state"/>
        <remap from="pid_output" to="ang_pid_output"/>
        
        <param name="p_term" type="double" value="1.75"/>
        <param name="i_term" type="double" value="0.0"/>
        <param name="d_term" type="double" value="1.0"/>
        
        <node pkg="lab1" name="anglePID" type="PID.py"/>
    </group>
    
     <group>
        <remap from="generic_setpoint" to="dist_setpoint"/>
        <remap from="current_state" to="dist_state"/>
        <remap from="pid_output" to="dist_pid_output"/>
        
        <node pkg="lab1" name="distancePID" type="PID.py"/>
    </group>

    <node pkg="balboa_core" name="balboa_serial" type="balboa_serial.py" />
    <node pkg="turtlesim" name="turtle_teleop" type="turtle_teleop_key" />
    <node pkg="lab1" name="input_capture" type="input_capture.py"/>
    <node pkg="lab1" name="convert_sensor_input" type="convert_sensor_input.py"/>
    <node pkg="lab1" name="adder" type="adder.py"/>

</launch>
